test/Libraries/RevitIntegrationTests/BugTests.cs

```diff
@@ -763,6 +763,24 @@ public void CanOpenAndRunOtherFilesAfterOpeningFileWithSelectNode()
             RunCurrentModel();
         }
 
+        [Test]
+        [Category("RegressionTests")]
+        [TestModel(@".\empty.rfa")]
+        public void AllElementsInActiveViewReturnsViewableElements()
+        {
+            string filePath = Path.Combine(workingDirectory, @".\Bugs\MAGN_7641_simplified.dyn");
+            string testPath = Path.GetFullPath(filePath);
+
+            ViewModel.OpenCommand.Execute(testPath);
+            AssertNoDummyNodes();
+            RunCurrentModel();
```

Comment:
Can we add Revit view manipulation to simulate the exact user scenario here before running it the second time?